example/src/main/java/com/factual/engine/braze/ExampleFactualClientReceiver.java

```diff
@@ -51,4 +53,13 @@ public void onConfigLoad(FactualConfigMetadata data) {
   public void onDiagnosticMessage(String diagnosticMessage) {
     Log.i("engine", diagnosticMessage);
   }
+
+  @Override
+  public void onCircumstancesMet(List<CircumstanceResponse> responses) {
+    for (CircumstanceResponse response : responses) {
+      if (response.getCircumstance().getActionId().equals("push-to-braze")) {
```

Comment:
- Should we check for a specific tag as well? 
- Do you know if `getActionId()` ever returns `null`?

Reply:
I clarify in the readme that they must set the action-id to "push-to-braze" for the example application to work.  I do this because the other test integrations have the same requirements to trigger so in testing responses will have the "push-to-airship" and "push-to-iterable" circumstances as well.